handlers.go

```diff
@@ -2107,22 +2108,22 @@ type adminStatsResponse struct {
 func (s *Server) handleAdminStats(c echo.Context) error {
 
 	var dealsTotal int64
-	if err := s.DB.Model(&contentDeal{}).Count(&dealsTotal).Error; err != nil {
+	if err := s.DB.Model(&util.ContentDeal{}).Count(&dealsTotal).Error; err != nil {
```

Comment:
If we are going to improve this, we should move the objects in a model.go file and reserve the util to utilities objects.

Reply:
ok, I will start with just content_deal for this PR only